packages/app-store/routing-forms/pages/route-builder/[...appPages].tsx

```diff
@@ -453,11 +454,13 @@ const Route = ({
                         }
                         if (option.value !== "custom") {
                           setRoute(route.id, {
-                            action: { ...route.action, value: option.value },
+                            action: { ...route.action, value: option.value, eventTypeId: option.eventTypeId },
```

Comment:
Now we store eventTypeId as well in routing to allow Rerouting to easily know which eventType is it rerouting to. Instead of it having to query by event slug.